2.3

ThomasLandauer opened the pull request.

ThomasLandauer committed:
Changed <meta http-equiv> to HTML5's <meta charset>

| Q             | A
| ------------- | ---
| Doc fix?      | [yes]
| New docs?     | [no]
| Applies to    | [2.3]
| Fixed tickets | []

ThomasLandauer committed:
Note about nesting blocks and {% endblock NAME %}

| Q             | A
| ------------- | ---
| Doc fix?      | [yes]
| New docs?     | [yes] 
| Applies to    | [2.3]
| Fixed tickets | []

ThomasLandauer committed:
Removed unnecessary(?) YAML hint

| Q             | A
| ------------- | ---
| Doc fix?      | [yes]
| New docs?     | [no]
| Applies to    | [2.3]
| Fixed tickets | []

I guess it's not necessary to do anything in __construct manually - the ArrayCollection is automatically initialized when you do php app/console doctrine:generate:entities AppBundle

Please double-check!

ThomasLandauer committed:
Fixed typo

| Q             | A
| ------------- | ---
| Doc fix?      | yes
| New docs?     | no
| Applies to    | 2.3
| Fixed tickets |

ThomasLandauer committed:
Another typo

ThomasLandauer committed:
Merge pull request #3 from ThomasLandauer/ThomasLandauer-patch-3

Removed unnecessary(?) YAML hint

ThomasLandauer committed:
Merge pull request #4 from ThomasLandauer/ThomasLandauer-patch-4

Fixed typo

ThomasLandauer committed:
Merge pull request #2 from ThomasLandauer/ThomasLandauer-patch-2

Note about nesting blocks and {% endblock NAME %}

ThomasLandauer committed:
Merge pull request #1 from ThomasLandauer/ThomasLandauer-patch-1

Changed <meta http-equiv> to HTML5's <meta charset>

WouterJ commented on book/doctrine.rst:
```diff
@@ -951,8 +951,6 @@ To relate the ``Category`` and ``Product`` entities, start by creating a
                 products:
                     targetEntity: Product
                     mappedBy: category
-            # don't forget to init the collection in the __construct() method
```
I think this comment is kind of usefull (and not everyone uses `doctrine:generate:entities`, even more it's better to never use this command).

ThomasLandauer replied:
OK, then please change it to:
# The ArrayCollection 'products' is automatically initialized in the __construct() method, when you call doctrine:generate:entities

WouterJ commented:
I like the other changes! (btw, can you please remove your merge commits from the commit history?)

ThomasLandauer commented:
Sorry, I'm new to GitHub and don't know where to delete what :-(
Took me a while to even figure out that I did not effectively create the Pull Request at the time I wrote the changes ;-)

stof commented:
@WouterJ you can squash the commits with the GH tool anyway, so this is not a hard requirement

WouterJ commented:
@ThomasLandauer no worries, I'll delete them when merging in that case.

ThomasLandauer commented:
Thanks :-)

ThomasLandauer committed:
Typo: Removed comma

| Q             | A
| ------------- | ---
| Doc fix?      | [yes]
| New docs?     | [no]
| Applies to    | [2.3]
| Fixed tickets |[]

ThomasLandauer committed:
Merge pull request #5 from ThomasLandauer/ThomasLandauer-patch-1

Typo: Removed comma

xabbuh commented on book/forms.rst:
```diff
@@ -97,7 +97,7 @@ from inside a controller::
 
             return $this->render('default/new.html.twig', array(
-                'form' => $form->createView(),
+                'form' => $form->createView()
```
This should be reverted to comply with the Symfony CS.

xabbuh commented on book/templating.rst:
```diff
@@ -370,6 +370,19 @@ When working with template inheritance, here are some tips to keep in mind:
       {% endblock %}
 
+* Blocks can be nested. For better overview, you can add the block name to the
+  ``{% endblock %}`` tag like so:
```
I would say "like this".

ThomasLandauer committed:
Update templating.rst

ThomasLandauer committed:
Added example for 'required' => false

| Q             | A
| ------------- | ---
| Doc fix?      | [yes]
| New docs?     | [no]
| Applies to    | [2.3]
| Fixed tickets | []

ThomasLandauer committed:
Fixed typo

| Q             | A
| ------------- | ---
| Doc fix?      | [yes]
| New docs?     | [no]
| Applies to    | [2.3]
| Fixed tickets | []

ThomasLandauer committed:
Fixed file path

"app/Resources/views/Form/fields.html.twig" didn't work for me - see next code block: "form_theme form 'AppBundle:Form:fields.html.twig'"

If this change is correct, I guess it needs to be corrected multiple times on this page.

ThomasLandauer committed:
Merge pull request #6 from ThomasLandauer/ThomasLandauer-patch-5

Fixed file path

ThomasLandauer committed:
Minor clarification

"the form" sounded like the HTML form.

| Q             | A
| ------------- | ---
| Doc fix?      | [yes]
| New docs?     | [no]
| Applies to    | [2.3]
| Fixed tickets | []

stof commented on cookbook/form/form_customization.rst:
```diff
@@ -282,7 +282,7 @@ can now re-use the form customization across many templates:
 .. code-block:: html+jinja
 
-    {# app/Resources/views/Form/fields.html.twig #}
+    {# src/AppBundle/Resources/views/Form/fields.html.twig #}
```
this looks wrong to me. Best practices are to put templates in ``app/Resources/views``

ThomasLandauer replied:
Anyhow. My point is that the path used in the code exampled throughout this page is inconsistent! I don't know what the best practice is. If you don't want to change it here, then please change it in the next codeblock (and probably some others).

WouterJ commented:
Hi @ThomasLandauer!

I've taken your commits and added a new commit to revert some changes (but I kept most of them!) and fixed the template name inconsistency. I put these commits in a new PR: https://github.com/symfony/symfony-docs/pull/6050

So I'm going to close this PR, but your contributions are not lost and you still get the credits. 

Thanks for helping us improving the documentation, I'm amazed you discovered these errors and took time to fix them! :christmas_tree:

ThomasLandauer commented:
Thanks :-)
